packages/dialtone-vue2/recipes/conversation_view/wysiwyg_editor/wysiwyg_editor.vue

```diff
@@ -0,0 +1,574 @@
+        importance="clear"
+        kind="inverted"
+        @click="onUnderlineTextToggle"
+      >
+        <template #icon>
+          <dt-icon
+            name="underline"
+            size="200"
+            class="d-fw-bold"
+          />
+        </template>
+      </dt-button>
+
+      <dt-button
+        v-if="showStrikeButton"
+        data-qa="dt-wysiwyg-editor-strike-btn"
+        class="h:d-bgc-black-300 d-fc-black-700 d-mx4"
+        :class="{ 'd-bgc-black-300 d-fc-black-900': $refs.richTextEditor?.editor?.isActive('strike') }"
+        size="sm"
+        importance="clear"
+        kind="inverted"
+        @click="onStrikethroughTextToggle"
+      >
+        <template #icon>
+          <dt-icon
+            name="strikethrough"
+            size="200"
+            class="d-fw-bold"
+          />
+        </template>
+      </dt-button>
+
+      <dt-button
+        v-if="showListItemsButton"
+        data-qa="dt-wysiwyg-editor-list-items-btn"
+        class="h:d-bgc-black-300 d-fc-black-700 d-mx4"
+        :class="{ 'd-bgc-black-300 d-fc-black-900': $refs.richTextEditor?.editor.isActive('bulletList') }"
+        size="sm"
+        importance="clear"
+        kind="inverted"
+        @click="onBulletListToggle"
+      >
+        <template #icon>
+          <dt-icon
+            name="list-bullet"
+            size="200"
+            class="d-fw-bold"
+          />
+        </template>
+      </dt-button>
+
+      <dt-button
+        v-if="showAddLink.showAddLinkButton"
+        data-qa="dt-wysiwyg-editor-add-link-btn"
+        class="h:d-bgc-black-300 d-fc-black-700 d-mx4"
+        size="sm"
+        importance="clear"
+        kind="inverted"
+        @click="openLinkInputModal"
+      >
+        <template #icon>
+          <dt-icon
+            name="link-2"
+            size="200"
+            class="d-fw-bold"
+          />
+        </template>
+      </dt-button>
+    </div>
+
+    <!-- Add/Remove link modal -->
+    <dt-modal
+      :show="showLinkInput"
+      :title="showAddLink.setLinkModalTitle"
+      :visually-hidden-close="true"
+      :visually-hidden-close-label="'Close link input modal'"
+      data-qa="dt-wysiwyg-editor-link-input-modal"
+      show-footer
+      kind="default"
+      size="default"
+      :close-button-props="{ ariaLabel: 'Close the dialog' }"
+      @update:show="(s) => updateInputModal(s)"
+    >
+      <div class="d-m4">
+        <dt-rich-text-editor
+          v-model="linkInput"
+          :input-aria-label="showAddLink.setLinkInputAriaLabel"
+          data-qa="dt-wysiwyg-editor-link-input"
+          :link="true"
+          :output-format="textOutputFormat"
+          :placeholder="setLinkPlaceholder"
+          input-class="d-bgc-black-100 d-bar4 d-ba d-baw1 d-bc-black-300 d-pl6 d-py4 d-ol-none"
+          @click="onInputFocus($event)"
+          @click.native.stop="onInputFocus($event)"
+          @focus="onInputFocus($event)"
+          @enter.native="setLink"
+        />
```

Comment:
Yes I tried using dt-input at first, but then I changed it because I wanted the link to be highlighted... do you think maybe I should go back to dt-input and forego the highlighting?